packages/lambdas/src/patient-import-query.ts

```diff
@@ -73,7 +71,7 @@ export async function handler(event: SQSEvent) {
   }
 }
 
-function parseBody(body?: unknown): PatientImportQueryBody {
+function parseBody(body?: unknown): ProcessPatientQueryEvemtPayload {
```

Comment:
for later: prob should try to reuse some of that

Reply:
100%

Reply:
Will handle in v1.